installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc

```diff
@@ -15,14 +15,42 @@ include::modules/ipi-install-extracting-the-openshift-installer.adoc[leveloffset
 
 include::modules/ipi-install-creating-an-rhcos-images-cache.adoc[leveloffset=+1]
 
-ifdef::upstream[]
-include::ipi-install-configuration-files.adoc[leveloffset=+1]
-endif::[]
+[id="ipi-install-configuration-files"]
+== Configuration files
+
+include::modules/ipi-install-configuring-the-install-config-file.adoc[leveloffset=+2]
+
+include::modules/ipi-install-setting-proxy-settings-within-install-config.adoc[leveloffset=+2]
+
+include::modules/ipi-install-modifying-install-config-for-no-provisioning-network.adoc[leveloffset=+2]
+
+include::modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc[leveloffset=+2]
+
+include::modules/ipi-install-configuring-managed-secure-boot-in-the-install-config-file.adoc[leveloffset=+2]
+
+include::modules/ipi-install-additional-install-config-parameters.adoc[leveloffset=+2]
+
+include::modules/ipi-install-bmc-addressing.adoc[leveloffset=+2]
 
-ifndef::upstream[]
-include::installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc[leveloffset=+1]
+include::modules/ipi-install-bmc-addressing-for-dell-idrac.adoc[leveloffset=+2]
+
+include::modules/ipi-install-bmc-addressing-for-hpe-ilo.adoc[leveloffset=+2]
+
+include::modules/ipi-install-bmc-addressing-for-fujitsu-irmc.adoc[leveloffset=+2]
+
+include::modules/ipi-install-root-device-hints.adoc[leveloffset=+2]
+
+include::modules/ipi-install-creating-the-openshift-manifests.adoc[leveloffset=+2]
+
+include::modules/ipi-install-configuring-ntp-for-disconnected-clusters.adoc[leveloffset=+2]
+
+include::modules/ipi-install-configure-network-components-to-run-on-the-control-plane.adoc[leveloffset=+2]
+
+ifeval::[{product-version} > 4.8]
```

Comment:
Unrelated to your change if this was already like this, but someone should try to get rid of this conditional. It won't show up in the preview (which caught me during my comparison) and it should just be removed from all 4.9+ branches.

Again, not for you to worry about, but just thinking out loud - not sure who best to ping on these docs.

Reply:
I do think he mentioned that they were trying to phase this out eventually. Just an FYI on this @johnwilkins if you do end up cleaning this up at some point.